src/components/updated-terms-conds/updated-terms-conds-controller.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response } from "express";
```

Comment:
there is no unit tests or integration tests for this - can we get them added

Reply:
Threw unit tests in, the integration tests have been a bit of a pain. May have to revisit them with another dev if we can get this PR sorted.